app/controllers/declaration/TransportDetailsController.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.declaration
+
+import config.AppConfig
+import controllers.actions.{AuthAction, JourneyAction}
+import controllers.declaration.routes.{SealController, SummaryPageController, TransportContainerController}
+import controllers.util.CacheIdGenerator.cacheId
+import forms.Choice.AllowedChoiceValues
+import forms.declaration.TransportDetails
+import handlers.ErrorHandler
+import javax.inject.Inject
+import models.requests.JourneyRequest
+import play.api.data.Form
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, Result}
+import services.CustomsCacheService
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.html.declaration.transport_details
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class TransportDetailsController @Inject()(
+  authenticate: AuthAction,
+  journeyAction: JourneyAction,
+  errorHandler: ErrorHandler,
+  customsCacheService: CustomsCacheService
+)(implicit ec: ExecutionContext, appConfig: AppConfig, override val messagesApi: MessagesApi)
+    extends FrontendController with I18nSupport {
+
+  implicit val countries = services.Countries.allCountries
+
+  def form(): Form[TransportDetails] = Form(TransportDetails.formMapping)
```

Comment:
The same here about form. Should be in TransportDetails companion object

Reply:
updated